### PR TITLE
Coronagraph fig

### PR DESCRIPTION
**Pull request by ehsteve**
This replaces the LASCO image.

### CODE REVIEW

File: figures/coronagraph_starfield.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+vv=Vizier(columns=['**'], row_limit=-1, column_filters={'Gmag':'<7'}, timeout=1200)
+vv.ROW_LIMIT = -1
+result = vv.query_region(search_coord, radius=4 * u.deg, catalog='I/345/gaia2')
+
+hpc_coords = []
+
+for this_object in result[0]:
+    tbl_crds = coordinates.SkyCoord(this_object['RA_ICRS'] * u.deg, this_object['DE_ICRS'] * u.deg, 1e12 * u.km, frame='icrs', obstime=map1.date)
+    hpc_coords.append(tbl_crds.transform_to(frames.Helioprojective(observer=map1.observer_coordinate)))
```
Comment (Cadair) [inline figures/coronagraph_starfield.py:32]: you don't actually need to do this transform. You can `ax.plot_coord` with the coordinate in ICRS and it should transform to the correct frame for you.

File: figures/coronagraph_starfield.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+# get the location of mars
+mars = get_body_heliographic_stonyhurst('mars', map1.date, observer=map1.observer_coordinate)
+mars_hpc = mars.transform_to(frames.Helioprojective(observer=map1.observer_coordinate))
```
Comment (Cadair) [inline figures/coronagraph_starfield.py:36]: same here.

File: figures/coronagraph_starfield.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+# now plot
+fig = plt.figure()
+ax1 = fig.add_subplot(1, 1, 1, projection=map1)
```
Comment (Cadair) [inline figures/coronagraph_starfield.py]: I find this cleaner, unless you actually need explicit access to the `fig` object.
```suggestion
ax1 = plt.subplot(projection=map1)
```

File: figures/coronagraph_starfield.py (hunk 1 above)
Comment (ehsteve) [reply figures/coronagraph_starfield.py:32]: You say that but I get an error unless I do this transform.

File: figures/coronagraph_starfield.py (hunk 2 above)
Comment (ehsteve) [reply figures/coronagraph_starfield.py:36]: Need to do this otherwise I get an error on `plot_coord`

File: figures/coronagraph_starfield.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+f = '2014_05_15__07_54_00_005__STEREO-A_SECCHI_COR2_white-light.jp2'
+
+path = os.path.join(data_dir, f)
+map1 = sunpy.map.Map(path)
+
+# finding the correct pointing location for the Sun as seen from STEREO
+sun = SkyCoord(0*u.deg, 0*u.deg, 0*u.km, obstime=map1.date,frame=frames.HeliographicStonyhurst, observer='Earth')
+stereo_icrs = map1.observer_coordinate.transform_to('icrs')
+sun_icrs = sun.transform_to('icrs')
```
Comment (Cadair) [inline figures/coronagraph_starfield.py]: You can replace this with a call to http://docs.astropy.org/en/stable/api/astropy.coordinates.get_body_barycentric.html#astropy.coordinates.get_body_barycentric

File: figures/coronagraph_starfield.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+# get the location of mars
+mars = get_body_heliographic_stonyhurst('mars', map1.date, observer=map1.observer_coordinate)
+mars_hpc = mars.transform_to(frames.Helioprojective(observer=map1.observer_coordinate))
+
+# now plot
+fig = plt.figure()
+ax1 = fig.add_subplot(1, 1, 1, projection=map1)
+
+map1.plot(axes=ax1, vmin=0, vmax=400)
+
```
Comment (Cadair) [inline figures/coronagraph_starfield.py:48]:
```python
lon, lat = ax1.coords
lon.set_major_formatter('d.dd')
lat.set_major_formatter('d.dd')
```

**Pull request merged**